cime_config/buildnml

```diff
@@ -313,13 +315,22 @@ def buildnml(case, caseroot, compname):
         if run_type == "hybrid" or run_type == "branch":
             compnames = ["clm4", "clm5", "clm2"]
             for comp in compnames:
-                clm_startfile = "%s.%s%s.r.%s-%s.nc" % (
-                    run_refcase,
-                    comp,
-                    inst_string,
-                    run_refdate,
-                    run_reftod,
-                )
+                if "PLUMBER2" in clm_usrdat_name:
+                    clm_startfile = "%s.%s%s.r.%s-%s.nc" % (
+                        run_refcase,
+                        comp,
+                        inst_string,
+                        run_refdate,
+                        start_tod,
+                    )
```

Comment:
Should RUN_REFTOD be set to START_TOD instead of using START_TOD here?

Reply:
@ekluzek Do you have any suggestions on this?

Reply:
In this context RUN_REFTOD is the time of day you want to use for the restart file to startup from. And START_TOD is the time of day you want the new run to start from. I can't think of a reason you'd ever want these to be different from each other. So using them interchangeably in one sense is OK, but it's maybe less clear on what's going on? In any case I don't think there is much of an issue with either way...

Reply:
Ok thanks for clarifying!

Reply:
Opening this back up, to just check on what both run_reftod and start_tod are going to be for PLUMBER2 cases? I take it at least start_tod is going to be non-zero -- or does that depend on the type of case? And will run_reftod be non-zero for the IC files for PLUMBER2 cases? It would be good to think this through to make sure we need this extra complexity in the code....

Reply:
Oops, sorry I didn't see this one since it was still resolved! The various PLUMBER cases (ad, postad, transient) all have RUN_REFTOD set to 00000, and START_TOD set to some nonzero value (eg, 54000)-- hence the decision to use START_TOD. We could update the shell commands to change RUN_REFTOD instead of START_TOD; I'd need to more thoroughly check that RUN_REFTOD is okay to change without other side effects, though.

Reply:
Ok, I added this to issue #2671